File: C++/Word-Codeforces problem.cpp
#include <iostream>
#include <cctype>
#include <string>
using namespace std;

string wordextension( string& word ){
    int length=word.size();
    int upper=0;
    int lower=0;
    for ( char& c:word){
        if(isupper(c)){
            upper++;
        }
        if (islower(c)){
            lower++;
        }

    }
    if(upper>lower){
        for ( char& u : word){
            u=toupper(u);
        }

    }
    else if(lower>=upper){
        for ( char& l : word){
            l=tolower(l);
        }
    }
    return word;
}
int main(){
    string userword;
    cin>> userword;
    cout<<wordextension(userword);
    return 0;
}

File: Python/Word codeforces problem.py
def word(something):
    x=len(something)
    up=0
    low=0
    for i in range(x):
        if something[i].isupper():
            up+=1
        elif something[i].islower():
            low+=1
        i+=1
    if(up>low):
        print(something.upper())
    elif(low>=up):
        print(something.lower())
something=str(input())
word(something)